### PR TITLE
KAA-993, KAA-1009, KAA-1010: Windows support

### PR DESCRIPTION
**Pull request by RostakaGmfun**
Based on feature/KAA-1010
/cc @dyosick

### CODE REVIEW

**Comment by iuriigolovach** [comment]
Can one of the admins verify this patch?

**Comment by rasendubi** [comment]
add to whitelist

File: client/client-multi/client-cpp/tools/build_sdk_thirdparty.bat [hunk 1]
```diff
@@ -35,70 +61,121 @@ goto :eof
...
+:buildSqlite
 
- cd %BUILD_HOME%
+  echo Building Sqlite...
+  call :download %SQLITE_SRC%.tar.gz %SQLITE_URL%
+  cd %SQLITE_SRC%
+  nmake /f Makefile.msc sqlite3.c
+  nmake /f Makefile.msc
+  mkdir %SQLITE_ROOT%\include
+  copy sqlite3.h %SQLITE_ROOT%\include
```
Comment (ghost) [inline client/client-multi/client-cpp/tools/build_sdk_thirdparty.bat:155]: There is one more header - sqlite3ext.h

**Comment by ghost** [comment]
Reviewed

**Comment by ghost** [comment]
+

**Pull request merged**